way_to_home/tests/unittests/place/testmodel.py

```diff
@@ -0,0 +1,111 @@
+"""
+Place model tests
+===============
+"""
+from decimal import Decimal
+
+from custom_user.models import CustomUser
+from django.test import TestCase, Client
+
+from place.models import Place
+
+
+class PlaceViewTest(TestCase):
+    """TestCase for providing Place model testing"""
+    def setUp(self):
+        """Method that provides preparation before testing Place model features."""
+        custom_user = CustomUser.objects.create(id=2, email='mymail@icloud.com', is_active=True)
+        custom_user.set_password('qwerty12345')
```

Comment:
Why do you use set_password() here? It does CustomUser.create() method.

Reply:
I think is a bad idea to use here custom methods. Because if you change the logic of your custom `create` method, u need to change this test also. And what if you remove this method at all?

Reply:
> Why do you use set_password() here? It does CustomUser.create() method.

When I pass a password to create method User is not created. You know how to fix this?

Reply:
> I think is a bad idea to use here custom methods. Because if you change the logic of your custom `create` method, u need to change this test also. And what if you remove this method at all?

Do you know a different way to do that?

Reply:
For example:
```
user = CustomUser(id=100, email='testuser@mail.com', is_active=True)
user.set_password('testpassword')
user.save()
```
But your way is also applicable. I think it is bad to only use our custom methods here.

Reply:
> > Why do you use set_password() here? It does CustomUser.create() method.
> 
> When I pass a password to create method, User is not created. You know how to fix this?

The user will be created, but his password will not be hashed.